Automattic/wp-calypso
# Comments: Sanitize content

#### Changes proposed in this Pull Request

This PR sanitizes the content of the comments displayed in the comment management section of Calypso (Site > Comments) so any unsafe markup is removed from the rendered content.

The content is sanitized by [the DOMPurify library](https://github.com/cure53/DOMPurify) which has been added as a new dependency.

#### Testing instructions

- Run `yarn` before starting Calypso, so the new `dompurify` dependency is installed.
- Navigate to Site > Comments (`/comments/all/:site`).
- Edit an existing comment and replace it with `"><img src=x onerror="alert(0);">`.
- Save the comment. 
- Make sure the JS alert is not displayed.
- Repeat with a variety of sites (Simple, Jetpack, Atomic).
- Make sure comments containing safe markup continue to work as expected.

Fixes 368-gh-Automattic/dotcom-manage
